src/routes/health.rs

```diff
@@ -1,5 +1,9 @@
 use axum::{http::StatusCode, routing::get, Router};
 
+static HEALTH_CHECK_TAG: &str = "health check";
+
+/// Used to determine, whether the API is online
+#[utoipa::path(get, path = "/live", tag = HEALTH_CHECK_TAG, responses((status = StatusCode::OK, description = "OK")))]
```

Comment:
This neglects the documenting of path /health. Does utoipa have a way of adding two doc paths to a single handler?

Reply:
I also wonder about the proper way to break the attributes here. This line is nearly too long, but rustfmt won't format it since it has no way of knowing how the developer of the attribute macro wanted it to be formatted.

Reply:
This is still unresolved, and since for the time being this aggregate mod::route() function is how I imagined building the routes tree, we need to be able to document all the routes inside of it.

Reply:
Unfortunately, only one `#path` macro can be used per function. This enforces creation of separate functions and documenting them.